package/MDAnalysis/coordinates/CRD.py

```diff
@@ -151,12 +153,23 @@ def __init__(self, filename, **kwargs):
         ----------
         filename : str or :class:`~MDAnalysis.lib.util.NamedStream`
              name of the output file or a stream
+
+        crdext   : bool (optional)
+             By default, noextended CRD format is used [``False``].
+             However, extended CRD format can be forced by
+             specifying `crdext` ``=True``.
         """
 
         self.filename = util.filename(filename, ext='crd')
         self.crd = None
 
-    def write(self, selection, frame=None):
+        # account for explicit crd format, if requested
+        if "crdext" in kwargs.keys():
+            self.crdext = kwargs.pop('crdext')
+        else:
+            self.crdext = False   # if not requested, default to NOEXT format
```

Comment:
Can be simplified as
```suggestion
        self.extended = kwargs.pop("extended", False)
```